app/controllers/api_docs/candidates_api_docs/open_api_controller.rb

```diff
@@ -0,0 +1,9 @@
+module APIDocs
+  module CandidatesAPIDocs
```

Comment:
Super nitty. I don't know why but `CandidateAPIDocs` feels right here. Interested to hear Duncans thoughts though i could def be wrong